test/controller/document_upload/document.upload.controller.spec.unit.ts

```diff
@@ -374,7 +374,7 @@ describe ("document.document_upload.controller tests", () => {
       throw {
         data: ["error"],
         message: "error",
-        status: 415,
+        status: HttpStatusCodes.UNSUPPORTED_MEDIA_TYPE,
```

Comment:
could you change this back to 415, the real error code will be a number , just want to make sure that when we do the comparison in the code, it works when comparing the number to the enum.

Reply:
just in the test i mean

Reply:
Ok chnaged back to "415".